setup/docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
     <<: *redash-service
     command: worker
     environment:
-      QUEUES: "queries"
+      QUEUES: "queries,schemas"
```

Comment:
It should be added to the scheduled worker and not the adhoc ones (no reason to block execution of adhoc queries for a background task).

Reply:
thx! i have fixed it.